scalableminds/webknossos
# Implement better error importing for async code

Due to #2773, I changed all the places where we do async stuff in the dashboard/admin views to handle errors better. Previously, errors within requests just froze the UI. Now, the loading state will be reset and an error will be shown to the user (either due to the messages sent by the server or a more generic one).

### URL of deployed dev instance (used for testing):
- https://betterformerrorhandling.webknossos.xyz

### Steps to test:
- go through all dashboard/admin views and test the async functionality (however, I think that a thorough review should be enough. most of the code didn't change much except for having a new try-catch wrapper).

### Issues:
- fixes #2773

------
